app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,87 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-services-and-info {
+  border-bottom: $govuk-focus-width solid govuk-colour("black");
+  padding-bottom: govuk-spacing(7);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: desktop) {
+    padding-bottom: govuk-spacing(6);
+  }
+}
+
+.homepage-services-and-info__list {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  list-style: none;
+  margin: 0 govuk-spacing(-3);
+  padding: 0;
+
+  @include govuk-media-query($from: "tablet") {
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr 1fr 1fr;
+  }
+
+  @include govuk-media-query($from: "desktop") {
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+  }
+}
+
+.chevron-card {
```

Comment:
Should `chevron-card` have its own sass file?

Reply:
Maybe when it's used by pages that aren't the homepage?

Reply:
Seems suitable. We can clear it up once we move this to the gem. Happy to call this resolved.